lib/internal/streams/utils.js

```diff
@@ -22,6 +22,17 @@ const kOnConstructed = Symbol('kOnConstructed');
 const kIsClosedPromise = SymbolFor('nodejs.webstream.isClosedPromise');
 const kControllerErrorFunction = SymbolFor('nodejs.webstream.controllerErrorFunction');
 
+const kState = Symbol('kState');
+const kObjectMode = 1 << 0;
+const kErrorEmitted = 1 << 1;
+const kAutoDestroy = 1 << 2;
+const kEmitClose = 1 << 3;
+const kDestroyed = 1 << 4;
+const kClosed = 1 << 5;
+const kCloseEmitted = 1 << 6;
+const kErrored = 1 << 7;
+const kConstructed = 1 << 8;
```

Comment:
How can we enforce that readable/writable stream will not have a bit state with the same position?

Reply:
Not sure. Suggestions?